static/style/blue/base.css

```diff
@@ -1039,8 +1039,7 @@ pre {
   margin-bottom: 1rem;
   background-color: transparent; }
   .table th, .ui-table-striped th,
-  .table td,
-  .ui-table-striped td {
+  .table td, .ui-table-striped td {
```

Comment:
is this whole file here by accident?

Reply:
Yes, thanks for catching this. I am not sure why this happens but it seems that make client listener might be running a different version.